autobump-maven/src/main/java/com/github/autobump/maven/model/MavenDependencyResolver.java

```diff
@@ -17,23 +18,44 @@
 
     @Override
     public Set<Dependency> resolve(Workspace workspace) {
+        Model model = getModel(workspace);
+        return model
```

Comment:
ik zou dit op 1 lijn zetten in plaats van 2
`return getModel(workspace)`...

Reply:
```suggestion
 return getModel(workspace)
                .getDependencies()
                .stream()
                .filter(dependency -> dependency.getVersion() != null)
                .map(dependency -> Dependency.builder()
                        .group(dependency.getGroupId())
                        .name(dependency.getArtifactId())
                        .version(getDependencyVersionFromModel(getModel(workspace), dependency.getVersion()))
                        .build())
                .collect(Collectors.toSet());
```

Dan moet ik wel 2x getModel doen... Tim is het beter om het dan toch zo te laten? Of haal ik het 2x op?